CMakeLists.txt

```diff
@@ -871,6 +871,8 @@ include_directories(Externals/picojson)
 
 add_subdirectory(Externals/rangeset)
 
+add_subdirectory(Externals/FatFs)
```

Comment:
missing the cmake file in the directory, appears to be causing other PRs to fail, which is weird.